pkg/sql/logictest/testdata/logic_test/rename_column

```diff
@@ -49,7 +49,7 @@ id name  species
 
 user testuser
 
-statement error pq: must be owner of table users or have CREATE privilege on table users
+statement error (pq: must be owner of table users or have CREATE privilege on table users)* (user testuser does not have CREATE privilege on relation users)*
```

Comment:
Is it hard to make these error messages match? Also I think the `|` would be better than the two `*`. RIght now, this will match `pq   `.